assets/scripts/region-redirects.js

```diff
@@ -17,24 +30,32 @@ document.addEventListener('DOMContentLoaded', () => {
     }
 });
 
+// returns the Datadog site associated with referrer URL, if applicable.
```

Comment:
we should add test coverage here once jest is fixed: https://datadoghq.atlassian.net/browse/WEB-2482